### PR TITLE
Method References 2] LPS-164448

### PR DESCRIPTION
**Pull request by Kvale1733**
@JanisZhang @tinatian Added test fix and resent
previous PR: https://github.com/liferay-core-infra/liferay-portal/pull/2489

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:reports

**Comment by JanisZhang** [comment]
@Kvale1733 Thanks Kevin!

File: modules/dxp/apps/portal-reports-engine-console/portal-reports-engine-console-test/src/testFunctional/tests/CPReportsadmin.testcase [hunk 1]
```diff
@@ -264,36 +264,36 @@ definition {
...
 
 		ReportsAdmin.viewReportEntry(reportsDefinitionName = "Reports Definition Name");
 
 		ReportsAdmin.viewReportEntryFields(
-			dataSource = "portal",
+			dataSource = "PORTAL",
 			reportsDefinitionDescription = "Reports Definition Description",
 			reportsDefinitionName = "Reports Definition Name",
-			reportsFileName = "reports_admin_template_source_sample_class_name.pdf",
+			reportsFileName = "reports_admin_template_source_sample_class_name.PDF",
```
Comment (tinatian) [inline modules/dxp/apps/portal-reports-engine-console/portal-reports-engine-console-test/src/testFunctional/tests/CPReportsadmin.testcase:284]: @JanisZhang The file name with ".PDF" is kind of weird, please check why we need to use upper case here, let's try to figure out a way to also accept lower case.

Tina

**Comment by tinatian** [comment]
@JanisZhang Please see my comment above and resend.

Thanks.
Tina

**Pull request closed without merging**